requirements-dev.txt

```diff
@@ -1,35 +1,98 @@
 #
```

Comment:
I think now would be a good time to drop the requirements files and store those deps in extras